### PR TITLE
Newq js interview

### PR DESCRIPTION
**Pull request by 23ashutosh3**
add js quistion

### CODE REVIEW

File: javascript/answers/23ashutosh3-q23.html [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+                <h4>
+                    <h4> Explain the working of timers in JavaScript? Also elucidate the drawbacks of using the timer, if any?</h4>
+                </h4>
+            </div>
+            <br>
+            <br>
+            <div>
+                <strong>Answer:</strong> 
+                <p>
+                    Timers are used to execute a piece of code at a set time or also to repeat the code in a given interval of time. This is done by using the functions setTimeout, setInterval and clearInterval.
```
Comment (shivamyth) [inline javascript/answers/23ashutosh3-q23.html:52]: Add an example.

**Review by shivamyth — CHANGES_REQUESTED** [review]

**Comment by 23ashutosh3** [comment]
Okk

**Comment by dakshkhetan** [comment]
@23ashutosh3 Please make the requested changes as this PR will be closed in 24 hours if no changes are made.

**Comment by 23ashutosh3** [comment]
I will do it today

**Comment by dakshkhetan** [comment]
Closing this PR, you can raise another one after making the requested changes.

**Pull request closed without merging**